docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -122,25 +118,37 @@ To manage the media controller features in your application, you must learn to s
 To manage the media controller features in your application, you must learn to handle requests from the client to the server:
 
 1. Create a `MediaControllerChangeRequestPlaybackInfoCallback` object (in [mobile](../../api/latest/device_api/mobile/tizen/mediacontroller.html#MediaControllerChangeRequestPlaybackInfoCallback), [wearable](../../api/latest/device_api/wearable/tizen/mediacontroller.html#MediaControllerChangeRequestPlaybackInfoCallback), and [tv](../../api/latest/device_api/tv/tizen/mediacontroller.html#MediaControllerChangeRequestPlaybackInfoCallback) applications) to define listeners for getting the requests from a client.
-  Each listener must invoke the appropriate function to perform the related update on the server and send an appropriate notification to the client. For example, the `onplaybackstaterequest` listener must use the `updatePlaybackState()` method to update the playback state.
+  Each listener must invoke the appropriate function to perform the related update on the server and send an appropriate notification to the client. For example, the `onshufflemoderequest` listener can change the playback shuffle mode of the media controller server.
 
    ```javascript
    var requestPlaybackInfoCb = {
-       onplaybackstaterequest: function(state, clientName) {
-           console.log('Request to change the playback state to: ' + state);
-           mcServer.updatePlaybackState(state);
+       onplaybackactionrequest: function(action, clientName) {
+<<<<<<< HEAD
```

Comment:
When you update this PR, please don't forget to resolve this merge conflict marker also.

Reply:
Done